viam-cartographer/src/carto_facade/carto_facade.cc

```diff
@@ -1017,10 +1017,7 @@ extern int viam_carto_init(viam_carto **ppVC, viam_carto_lib *pVCL,
         return VIAM_CARTO_LIB_INVALID;
     }
     // check that IMU is correctly set up
-    if ((ac.use_imu_data == true &&
-         biseqcstr(c.movement_sensor, ("")) == true) ||
-        (ac.use_imu_data == false &&
-         biseqcstr(c.movement_sensor, ("")) == false)) {
```

Comment:
A configured movement sensor that can not be used as an IMU or odometer will not cause cartographer to error out anymore.